ci/setup-new-buildkite-agent/setup-new-machine.sh

```diff
@@ -21,7 +21,7 @@ EOF
 
 apt install -y build-essential pkg-config clang cmake sysstat linux-tools-common \
   linux-generic-hwe-18.04-edge linux-tools-generic-hwe-18.04-edge \
```

Comment:
just noticed this 18.04 here. maybe should change this to 20.04? haven't tested that yet